packages/blade/src/components/Typography/Heading/Heading.stories.tsx

```diff
@@ -3,13 +3,56 @@ import {
   Title as StorybookTitle,
   Subtitle,
   Primary,
-  Props,
+  ArgsTable,
   Stories,
   PRIMARY_STORY,
 } from '@storybook/addon-docs';
+import { Highlight, Link } from '@storybook/design-system';
+import type { ReactElement } from 'react';
+import useMakeFigmaURL from '../../../_helpers/storybook/useMakeFigmaURL';
 import type { HeadingProps } from './';
 import HeadingComponent from './';
 
+const Page = (): ReactElement => {
```

Comment:
need to create this as a functional component since can't use hooks outside functional components, even though the storybook was rendering it as a component internally but eslint can't figure it out since it was rendered as an object's property.